### ISSUE TITLE
Couple of strange bugs

### ISSUE THREAD

**Issue by Tatertot10** [Tsuey/L4D2-Community-Update#397]
### Description

First bug is that when i start in a map the sound gets all static but i change the sound options to headphones and its fine untill i load into the next map. Second one is theres no saferoom music (i miss the music lol). Last bug is that on the passing map 1 the sound of the storm starts when i first load in and ends untill i beat it so all i can hear is the wind and rain untill i get in a indoor area.

**Comment by alexiscoutinho** [comment]
Do you play with mods? Have you tried verifying the integrity of your game files?

**Comment by Tatertot10** [comment]
Yeah did both and bothing work. Finally found other people online who had similar problems but no one has found a fix.

**Comment by alexiscoutinho** [comment]
Try playing without mods and run `snd_rebuildaudiocache`. When the cmd stops doing its thing, restart the game.
